config/printer-biqu-bx-2021.cfg

```diff
@@ -0,0 +1,390 @@
+# This file contains common pin mappings for the BigTreeTech SKR SE BX.
+# To use this config, the firmware should be compiled for the
+# STM32H743 with a "128KiB bootloader". Additionally, GPIO pins PB5
+# and PE5 need to be set at microcontroller startup.
+
+######################################################################
+# NOTE: In order to use the TFT70-BX display, you MUST add PB5 and PE5
+# to the `GPIO pins to set at micro-controller startup` section when
+# running "make menuconfig"
+######################################################################
+
+# The "make flash" command does not work on the SKR SE BX. Instead,
+# after running "make", copy the generated "out/klipper.bin" file to a
+# file named "firmware.bin" on an SD card and then restart the SKR SE BX
+# with that SD card. After klipper has been flashed once to the board,
+# you can update klipper by leaving a microSD inserted and running the
+# scripts/flash-sd.sh script.
+
+# See docs/Config_Reference.md for a description of parameters.
+
+########################################
+# Steppers
+########################################
+
+[stepper_x]
+step_pin: PG13
+dir_pin: !PG12
+enable_pin: !PG14
+microsteps: 16
```

Comment:
You do not use interpolate, which is good acc. to https://github.com/Klipper3d/klipper/pull/4977
Why this low `microsteps`?

Reply:
This actually does use interpolate, the default is True.

Reply:
Acc to #4977 it seems preferable to use higher `microsteps` and set `interpolate: false`
The BIQU BX SKR SE-BX should not even break a sweat with this

Reply:
While the BIQU BX definitely doesn't have a problem with this config, testing shows that the printer is crazy loud using the spreadcycle mode. For default, I decided to leave it as stealthchop because most users will find it annoying and want to quiet it down anyways.